.husky/pre-commit

```diff
@@ -1,2 +1,2 @@
 pnpx lint-staged
-pnpm test -- --onlyChanged
\ No newline at end of file
+pnpm test -- --onlyChanged
```

Comment:
And back again? Permissions too.

Either way is fine, but probably best to keep it stable

Reply:
I think i've _finally_ got ontop of my husky dramas.  NVM has been meddling with it